eiffel-vocabulary/EiffelServiceStartedEvent.md

```diff
@@ -0,0 +1,21 @@
+
+# EiffelServiceStartedEvent (ServS)
```

Comment:
I think we should have abbreviations that are unambiguous if possible. ServS can be interpreted as ServiceStopped. Perhaps SStart and SStop is better. Also that would lead to SRes and SPaus.